proto/canine_chain/rns/query.proto

```diff
@@ -16,100 +16,87 @@ option go_package = "github.com/jackalLabs/canine-chain/x/rns/types";
-    option (google.api.http).get = "/jackal-dao/canine-chain/rns/names";
+  rpc AllNames(QueryAllNames) returns (QueryAllNamesResponse) {
+    option (google.api.http).get = "/jackal/canine-chain/rns/names";
   }
 
   // Queries a Bid by index.
-  rpc Bids(QueryBidRequest) returns (QueryBidResponse) {
-    option (google.api.http).get = "/jackal-dao/canine-chain/rns/bids/{index}";
+  rpc Bid(QueryBid) returns (QueryBidResponse) {
+    option (google.api.http).get = "/jackal/canine-chain/rns/bids/{index}";
   }
 
   // Queries a list of Bids.
-  rpc BidsAll(QueryAllBidsRequest) returns (QueryAllBidsResponse) {
-    option (google.api.http).get = "/jackal-dao/canine-chain/rns/bids";
+  rpc AllBids(QueryAllBids) returns (QueryAllBidsResponse) {
+    option (google.api.http).get = "/jackal/canine-chain/rns/bids";
   }
 
   // Queries a Listing by index.
-  rpc Forsale(QueryForsaleRequest) returns (QueryForsaleResponse) {
+  rpc ForSale(QueryForSale) returns (QueryForSaleResponse) {
     option (google.api.http).get =
-        "/jackal-dao/canine-chain/rns/forsale/{name}";
+        "/jackal/canine-chain/rns/for_sale/{name}";
   }
 
   // Queries all Listings.
-  rpc ForsaleAll(QueryAllForsalesRequest) returns (QueryAllForsalesResponse) {
-    option (google.api.http).get = "/jackal-dao/canine-chain/rns/forsale";
+  rpc AllForSale(QueryAllForSale) returns (QueryAllForSaleResponse) {
+    option (google.api.http).get = "/jackal/canine-chain/rns/for_sale";
   }
 
   // Queries a Init by index.
-  rpc Init(QueryInitRequest) returns (QueryInitResponse) {
+  rpc Init(QueryInit) returns (QueryInitResponse) {
     option (google.api.http).get =
-        "/jackal-dao/canine-chain/rns/init/{address}";
+        "/jackal/canine-chain/rns/init/{address}";
   }
 
   // Queries a list of Init items.
-  rpc InitAll(QueryAllInitsRequest) returns (QueryAllInitsResponse) {
-    option (google.api.http).get = "/jackal-dao/canine-chain/rns/init";
+  rpc AllInits(QueryAllInits) returns (QueryAllInitsResponse) {
+    option (google.api.http).get = "/jackal/canine-chain/rns/init";
   }
 
   // Queries a list of ListOwnedNames items.
-  rpc ListOwnedNames(QueryListOwnedNamesRequest)
+  rpc ListOwnedNames(QueryListOwnedNames)
       returns (QueryListOwnedNamesResponse) {
     option (google.api.http).get =
-        "/jackal-dao/canine-chain/rns/list_owned_names/{address}";
+        "/jackal/canine-chain/rns/list_owned_names/{address}";
   }
 }
 
-// QueryParamsRequest is request type for the Query/Params RPC method.
-message QueryParamsRequest {}
+// QueryParams is request type for the Query/Params RPC method.
+message QueryParams {}
 
 // QueryParamsResponse is response type for the Query/Params RPC method.
 message QueryParamsResponse {
   // params holds all the parameters of this module.
   Params params = 1 [ (gogoproto.nullable) = false ];
 }
 
-message QueryWhoisRequest { string index = 1; }
+message QueryName { string index = 1; }
```

Comment:
should be int not string

Reply:
should be string ;P shouldn't be called index since it's not an index. Leftovers from ignite :/